pkg/controllermanager/apis/config/v1alpha1/defaults.go

```diff
@@ -137,6 +137,10 @@ func SetDefaults_ControllerManagerConfiguration(obj *ControllerManagerConfigurat
 		v := DefaultControllerConcurrentSyncs
 		obj.Controllers.ShootMaintenance.ConcurrentSyncs = &v
 	}
+	if obj.Controllers.ShootMaintenance.EnableShootControlPlaneRestarter == nil {
+		b := true
+		obj.Controllers.ShootMaintenance.EnableShootControlPlaneRestarter = &b
```

Comment:
```suggestion
		obj.Controllers.ShootMaintenance.EnableShootControlPlaneRestarter = pointer.Bool(true)
```

Reply:
I used your proposed approach before and got:
```
errors in package "github.com/gardener/gardener/pkg/controllermanager/apis/config/helper":
the following imports did not match any allowed prefix:
  k8s.io/utils/pointer
```

https://prow.gardener.cloud/view/gs/gardener-prow/pr-logs/pull/gardener_gardener/8282/pull-gardener-unit/1687388756874629120

Therefore, I went with this instead. Do you propose to add it to the allowed imports instead?

Reply:
Lol :) 
Yes, I think we can add this pointer utils package to the allowed list

Reply:
https://github.com/gardener/gardener/pull/8282/commits/a179c934c55c137ae048773798a8ab7db3829518 adds the necessary change.